jbpm-wb-executor-service/jbpm-wb-executor-service-client/src/main/java/org/jbpm/workbench/es/client/editors/requestlist/RequestListPresenter.java

```diff
@@ -97,12 +102,16 @@ public RequestListPresenter() {
     public RequestListPresenter(RequestListViewImpl view,
                                 Caller<ExecutorService> executorServices,
                                 DataSetQueryHelper dataSetQueryHelper,
-                                Event<RequestChangedEvent> requestChangedEvent
+                                Event<RequestChangedEvent> requestChangedEvent,
+                                Event<JobSelectedEvent> jobSelectedEvent,
+                                PlaceManager placeManager
     ) {
         this.view = view;
         this.executorServices = executorServices;
         this.dataSetQueryHelper = dataSetQueryHelper;
         this.requestChangedEvent = requestChangedEvent;
+        this.jobSelectedEvent= jobSelectedEvent;
```

Comment:
Perhaps run a code reformat in this class, there are a few places in this class that need fixing.